src/main/kotlin/com/openlattice/shuttle/source/LocalFileOrigin.kt

```diff
@@ -0,0 +1,24 @@
+package com.openlattice.shuttle.source
+
+import org.slf4j.LoggerFactory
+import java.io.InputStream
+import java.nio.file.Files
+import java.nio.file.Path
+
+data class LocalFileOrigin(val path: Path, val filter: (Path) -> Boolean = { true } ) : IntegrationOrigin() {
```

Comment:
(a) Does this need to move?
(b) We shouldn't support this origin for shuttle server (only shuttle cli)